examples/jsm/nodes/accessors/TextureNode.js

```diff
@@ -46,13 +47,17 @@ class TextureNode extends UniformNode {
 
 	}
 
-	getDefaultUV( uvNode = null ) {
+	getDefaultUV() {
 
-		const texture = this.value;
+		return uv( this.value.channel );
+
+	}
 
-		if ( uvNode === null ) uvNode = uv( texture.channel );
+	getTextureMatrix( uvNode ) {
```

Comment:
Maybe `getTransformedUV` or `getFinalUV`? Because this method doesn't return the matrix.

Reply:
`getTransformedUV()` sounds good to me 👍

Reply:
Yep!